### PR TITLE
Tias bps

### PR DESCRIPTION
**Pull request by ekr**
This is an update of Fippo's

### CODE REVIEW

**Comment by ekr** [comment]
See PR#286

**Comment by fluffy** [comment]
LGTM

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -3806,13 +3806,14 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+                            <t>TIAS = AS * 1000 * 0.95 - 50 * 40 * 8</t>
 
                         </list>
 
                         The 50 is based on 50 packets per second, the 40
-                        is based on an estimate of total header size,
-                        and the 0.95 is to allocate 5% to RTCP. If more
+                        is based on an estimate of total header size, the
+                        1000 changes the unit from kbps to bps (as required
+                        by TIAS) and the 0.95 is to allocate 5% to RTCP. If more
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: add oxford comma

**Comment by juberti** [comment]
lgtm with comma

**Pull request merged**